setup/index.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
+const core = require('@actions/core');
+const tc = require('@actions/tool-cache');
+const { getDownloadObject } = require('./lib/utils');
+const fs = require('fs');
+const mv = require('mv');
+const { version } = require('os');
+const execSync = require('child_process').execSync;
```

Comment:
Looks like these two are not used. We can remove them.

Reply:
Removed with the latest commit, thanks for catching that. :)